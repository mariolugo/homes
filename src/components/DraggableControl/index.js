import React, { useState } from 'react';
import styled from 'styled-components';

const ControlStyled = styled.div`
  height: 35px;
  box-sizing: border-box;
  width: 290px;
  border-radius: 2px;
  background-color: rgb(255, 255, 255);
  box-shadow: rgba(0, 0, 0, 0.5) 0px 1px 2px 0px;
  margin-bottom: 15px;
  cursor: pointer;
  position: absolute;
  bottom: 20px;
  left: 50%;
  margin-left: -148px;
  padding: 8px;
  span {
    font-size: 14px;
  }
`;

const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
  // Hide checkbox visually but remain accessible to screen readers.
  // Source: https://polished.js.org/docs/#hidevisually
  border: 0;
  clip: rect(0 0 0 0);
  clippath: inset(50%);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  white-space: nowrap;
  width: 1px;
`;

const Icon = styled.svg`
  fill: none;
  stroke: white;
  stroke-width: 2px;
`;

const StyledCheckbox = styled.div`
  display: inline-block;
  width: 16px;
  height: 16px;
  border: 1px solid black;
  background: ${(props) => (props.checked ? 'rgb(141, 122, 255)' : 'white')};
  border-radius: 3px;
  transition: all 150ms;
  margin-right: 10px;
  ${HiddenCheckbox}:focus + & {
    box-shadow: 0 0 0 3px rgb(141, 122, 255);
  }
  ${Icon} {
    visibility: ${(props) => (props.checked ? 'visible' : 'hidden')};
  }
`;

const CheckboxContainer = styled.div`
  display: inline-block;
  vertical-align: middle;
`;

/**
 * Component used for the map, in here we have a checkbox and we can add logic to handle the checkbox.
 * For example we can activate the checkbox and filter the homes if the map changes
 */
const DraggableControl = () => {
  const [active, setActive] = useState(false);

  const onCheckboxChange = () => {
    setActive((prev) => !prev);
  };

  return (
    <ControlStyled>
      <div>
        <form>
          <label htmlFor="checkbox">
            <CheckboxContainer>
              <HiddenCheckbox id="checkbox" checked={active} onChange={onCheckboxChange} />
              <StyledCheckbox checked={active}>
                <Icon viewBox="0 0 24 24">
                  <polyline points="20 6 9 17 4 12" />
                </Icon>
              </StyledCheckbox>
            </CheckboxContainer>
            <span>Filtrar mientras te mueves en el mapa</span>
          </label>
        </form>
      </div>
    </ControlStyled>
  );
};

export default DraggableControl;
