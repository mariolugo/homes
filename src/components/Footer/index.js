import React from 'react';
import styled from 'styled-components';
import { Col, Row } from 'react-styled-flexboxgrid';
import { WrapperGrid } from '../';
import FooterLogo from '../../assets/icons/footer-logo.svg';
import Messenger from '../../assets/icons/messenger.svg';

const FooterStyled = styled.div`
  background-color: white;
  border-top: 1px solid ${({ theme }) => theme.colors.grayLight};
  margin: 0px auto;
  min-height: 60px;

  @media only screen and (max-width: 600px) {
    padding-top: 10px;
  }
`;

const LinkStyled = styled.p`
  font-size: 13px;
`;

const FooterLink = styled(LinkStyled)`
  text-align: center;
`;

const ListStyled = styled.ul`
  list-style: none;
  padding-left: 0px;
`;

const ListItemStyled = styled.li`
  svg {
    display: inline-block;
    margin-right: 5px;
    position: relative;
    top: 2px;
  }
  p {
    display: inline-block;
    margin: 0;
  }

  @media only screen and (max-width: 600px) {
    text-align: center;
  }
`;

const LogoContainer = styled(Col)`
  display: flex;
  padding-right: 0;
  align-items: center;
  justify-content: center;
`;

const ColCentered = styled(Col)`
  display: flex;
  align-items: center;
  justify-content: center;
`;

/**
 * Main footer of the application
 */
const Footer = () => (
  <FooterStyled>
    <WrapperGrid>
      <Row>
        <Col xs={12} sm={4}>
          <Row>
            <LogoContainer xs={12} sm={3}>
              <FooterLogo />
            </LogoContainer>
            <Col xs={12} sm={9}>
              <ListStyled>
                <ListItemStyled>
                  <LinkStyled>hey@homie.mx</LinkStyled>
                </ListItemStyled>
                <ListItemStyled>
                  <LinkStyled>5015 1546</LinkStyled>
                </ListItemStyled>
                <ListItemStyled>
                  <Messenger />
                  <LinkStyled>Escríbenos por facebook</LinkStyled>
                </ListItemStyled>
              </ListStyled>
            </Col>
          </Row>
        </Col>
        <Col xs={12} sm={3}>
          <ListStyled>
            <ListItemStyled>
              <LinkStyled>Te buscamos depa</LinkStyled>
            </ListItemStyled>
            <ListItemStyled>
              <LinkStyled>Blog</LinkStyled>
            </ListItemStyled>
            <ListItemStyled>
              <LinkStyled>Equipo</LinkStyled>
            </ListItemStyled>
          </ListStyled>
        </Col>
        <ColCentered xs={12} sm={2}>
          <FooterLink>Aviso de privacidad</FooterLink>
        </ColCentered>
        <ColCentered xs={12} sm={3}>
          <FooterLink>Términos y condiciones</FooterLink>
        </ColCentered>
      </Row>
    </WrapperGrid>
  </FooterStyled>
);

export default Footer;
