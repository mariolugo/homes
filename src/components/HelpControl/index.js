import React from 'react';
import styled from 'styled-components';
import Image from 'next/image';

const ControlStyled = styled.div`
  width: 50px;
  height: 50px;
  background-color: white;
  cursor: pointer;
  position: absolute;
  bottom: 20px;
  right: 50%;
  margin-right: 10px;
  box-sizing: border-box;
  box-shadow: rgba(0, 0, 0, 0.5) 0px 1px 2px 0px;
  padding: 13px 9px;
  border-radius: 2px;
`;

/**
 * Help control that is required in the map.
 */
const HelpControl = () => {
  return (
    <ControlStyled>
      <Image src="/images/help_map.png" alt="Help Map" width={35} height={35} />
    </ControlStyled>
  );
};

export default HelpControl;
