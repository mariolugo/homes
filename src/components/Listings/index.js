import React from 'react';
import styled from 'styled-components';
import { Col, Row } from 'react-styled-flexboxgrid';
import { Post } from '../';

const ListingsContainerStyled = styled.div``;

const ListingsContainer = ({ posts = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15] }) => (
  <ListingsContainerStyled>
    <Row>
      {posts.length > 0 &&
        posts.map((p) => (
          <Col key={p} xs={12} md={6}>
            <Post />
          </Col>
        ))}
    </Row>
  </ListingsContainerStyled>
);

export default ListingsContainer;
