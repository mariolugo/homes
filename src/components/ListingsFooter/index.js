import React from 'react';
import styled from 'styled-components';

const ListingsFooterStyled = styled.div`
  p {
    color: ${({ theme }) => theme.colors.textPrimary};
  }
`;

const Title = styled.p`
  text-align: center;
  font-family: 'Open Sans Bold', sans-serif;
`;

const Subtitle = styled.p`
  text-align: center;
`;

const Button = styled.button`
  border-radius: 2px;
  border: none;
  width: 300px;
  display: block;
  height: 35px;
  background-color: ${({ theme }) => theme.colors.secondary};
  box-shadow: rgba(0, 0, 0, 0.5) 0px 1px 2px 0px;
  color: white;
  font-family: 'Open Sans Bold', sans-serif;
  line-height: 19px;
  padding: 8px 0px;
  margin: 20px auto;
`;

const ListingsFooter = () => (
  <ListingsFooterStyled>
    <Title>¿No encuentras inmueble?</Title>
    <Subtitle>Dinos qué quieres y nosotros te lo buscamos</Subtitle>
    <Button>Encuéntrenme un inmueble</Button>
  </ListingsFooterStyled>
);

export default ListingsFooter;
