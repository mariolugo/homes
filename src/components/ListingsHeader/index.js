import React from 'react';
import styled from 'styled-components';
import { Col, Row } from 'react-styled-flexboxgrid';
import Filter from '../../assets/icons/filter.svg';

const ListingHeaderStyled = styled.div`
  width: 100%;
  margin-bottom: 15px;
`;

const FilterButton = styled.button`
  height: 50px;
  background-color: ${({ theme }) => theme.colors.primary};
  border: none;
  border-radius: 3px;
  color: white;
  width: 100%;
  cursor: pointer;
  box-shadow: rgba(0, 0, 0, 0.5) 0px 1px 2px 0px;

  svg {
    display: inline-block;
    margin-right: 5px;
    position: relative;
    top: 2px;
  }

  p {
    display: inline-block;
    font-family: 'Open Sans Bold', sans-serif;
    font-size: 16px;
    margin: 0;
  }

  @media only screen and (max-width: 500px) {
    margin-top: 15px;
  }
`;

const InfoBox = styled.div`
  background-color: white;
  height: 50px;
  max-height: 50px;
  border-radius: 3px;
  box-shadow: rgba(0, 0, 0, 0.5) 0px 1px 2px 0px;

  h3 {
    font-size: 16px;
    line-height: 1;
    padding: 7px 0px 0px;
    margin: 0px 0px 0px 10px;
    color: ${({ theme }) => theme.colors.primary};
    font-family: 'Open Sans Bold', sans-serif;
  }

  h4 {
    font-size: 14px;
    line-height: 1;
    padding: 7px 0px 0px;
    margin: 0px 0px 0px 10px;
    color: ${({ theme }) => theme.colors.textPrimary};
  }
`;

const ListingHeaders = () => (
  <ListingHeaderStyled>
    <Row>
      <Col xs={12} sm={9}>
        <InfoBox>
          <h3>Ciudad de Mexico</h3>
          <h4>230 departamentos</h4>
        </InfoBox>
      </Col>
      <Col xs={12} sm={3}>
        <FilterButton>
          <Filter />
          <p>Filtros</p>
        </FilterButton>
      </Col>
    </Row>
  </ListingHeaderStyled>
);

export default ListingHeaders;
