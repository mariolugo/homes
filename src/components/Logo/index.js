import React from 'react';
import Image from 'next/image';

/**
 * Main logo of the app
 */
const Logo = () => <Image src="/images/logo.png" alt="Logo" width={94} height={32} />;

export default Logo;
