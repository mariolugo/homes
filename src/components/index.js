import NavBar from './NavBar';
import Listings from './Listings';
import Map from './Map';
import Footer from './Footer';
import Post from './Post';
import WrapperGrid from './WrapperGrid';

export { NavBar, Listings, Map, Footer, Post, WrapperGrid };
