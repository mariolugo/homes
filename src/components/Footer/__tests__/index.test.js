import { mountWithTheme } from '../../../../tests/themeProvider';
import { Footer } from '../../../components';

/** @test {Footer Component} */
describe('Footer Component', () => {
  it('should render without crashing', () => {
    const wrapper = mountWithTheme(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
