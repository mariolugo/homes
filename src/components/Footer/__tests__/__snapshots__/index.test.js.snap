// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bqyKva jQYkRD"
      >
        <div
          class="sc-bdfBwQ sc-kLgntA guSBMN iXAtuJ"
        >
          <div
            class="sc-gsTCUz ikQfAY"
          >
            <div
              class="sc-dlfnbm eSUTag"
            >
              <div
                class="sc-gsTCUz ikQfAY"
              >
                <div
                  class="sc-dlfnbm sc-bkzZxe hziHjY itmDFr"
                >
                  <svg
                    class="Footer__Logo-h84djn-5 cnyeeA"
                    height="35"
                    width="35"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M35 0l-.041 35H20.43v-4.503h10.03V4.503H4.58v9.006H0V0h35zM15.69 16.897V35h-4.518v-7.155H4.518V35H0V16.897h4.518v6.853h6.654v-6.853h4.518z"
                      fill="#FF0364"
                    />
                  </svg>
                </div>
                <div
                  class="sc-dlfnbm fLHlfI"
                >
                  <ul
                    class="sc-fodVxV cTyRtG"
                  >
                    <li
                      class="sc-fFubgz jmgknO"
                    >
                      <p
                        class="sc-kstrdz iUronQ"
                      >
                        hey@homie.mx
                      </p>
                    </li>
                    <li
                      class="sc-fFubgz jmgknO"
                    >
                      <p
                        class="sc-kstrdz iUronQ"
                      >
                        5015 1546
                      </p>
                    </li>
                    <li
                      class="sc-fFubgz jmgknO"
                    >
                      <svg
                        height="13"
                        width="12"
                        xmlns="http://www.w3.org/2000/svg"
                        xmlns:xlink="http://www.w3.org/1999/xlink"
                      >
                        <image
                          fill="none"
                          fill-rule="evenodd"
                          height="13"
                          transform="translate(0 -1)"
                          width="12"
                          xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAgCAYAAAAFQMh/AAAABGdBTUEAALGOfPtRkwAABM1JREFUSA3Nll1om2UUx5M3CVk+mtXZimUdrC10Ewdzdl4EhJalNu2uFCbeqExldkOHguKFF1JQwYlUUClWrSgIXlRFJrRpta5ICupa2DoKazFdSLPVdbrZNk2bb38nyxvepG+a2jLwwMl5nvOc8/8/n+eN0bBZ6epS9u055IgpNQ6XKVEVN5srDGaDwRJPRdIJ841kOr4yPffTiqGrK70ZSGO5oJaWFuANTrPZ7DKZTI2KotQbjcaHM5nMfvySP43+kk6nZ1Op1EwymVyiHxkdHU1iS8pGxIrX662E4ACEj4HQhjZAatVDIy6GP4D+SPv7SCRy0e/3L9LX3QFdYla5w2Kx7EdfBORxyFx6ZKV85CyR088OfBiLxaZZ/VpxrKnY0dbW5oDwUba2lzFPqRUW52n7uZwHsY+AM19fX38lEAgktDEFxG632+ZwOE6Q8AFB92gDt9IGZxd6lNyb1dXVk+FwOH/ueWK5RE6n8ymC3id4x1aISuRY8B+xWq1ztbW1l4LBYPbMFTXYZrMd5sa+A6nu5VHjtmIFU7CFQ83PrripqcnOFn9KwAF14A5YO/h7XS7Xt/Pz84nsitn/doiO3AGyAkhuuwcurziF2MRMnkF1n1ZB5u1OCnML1X2fmvg0RCv0JT4rwoE8S8dk5Pns4cpP0qm8Pbzhr4BdRHsBeJVIKSjrJiyEuOcYH6d9lPYuDeotqttBhaok51qWFIAMcTOAPO/z+eSNd9IO5vxZXNpraBjtpXicxtYUkUrcXcKpMCg1d0MhRkgD1OMTAwMD4xI8ODj4M+6TAIewMfQG7W8gPBaPx7sBf4W+Rw8Y/33ccuWQ3mCRLwzpKVbq5wXIu8wK5MMQnqbzA/Y4k3oa0gBv9mN8cmF1hdiDxvb29o8gf0E3AidB1zAnFxYWfBUVFTsBfYD6e4H6+5cmRy5puqOjoxrbw4qOacb0mj2ScF5vJOe7jn2J1Y5VVVU1UwA+YQvP2u32t6h02nuR9ng8d0PYvQlSA3i/K/xcZlX5GiqE9EX+Rl/nBv4hVQfC7wCVz6MN7WQSb6rkYtmJt/E/iZaTFDjTxtbW1p18jSboNGgyopD20F/Gfwp7r2Ys3yTmPSbezcReI+7l/MAGDXICiUSiyTQ7OxtrbGzcS6xbEy8PfzdgclZOjb+46SbmIZxPYNe95+Jg6RP25dDQ0Fk5Y9nzL5jJTWnnpJKAfWqnlBUypFl+SsVo/cIhXOLLEvNMJsntY6DgrLVJ223nsD8XrjyxNKLR6LuY39ByNVjC/6sI5q+rq6tn1MTsZ1E6fKCjDQ0NE1yUFrpSW7O7gd2upFjtNFv83MjIyBUVLE8sDv4XXa+rqxtn2w+jNWrQVi2Ea+Bcoox2Dg8PT2hxdC8FFaiZoHMk6Y5rAfTaufNcYsyHvkFplb+9BSJ/1tcJ2/IPBUM+DHliActNpGCXNMny3Y2hUVkl8X2hUKh/amoqronJN3WJOWft+QqgzP4yKpOpBVT+DKqTykAkn8Or+C5gfcvLy+fGxsaW6ZcUXWI1GhAhnMF+BvjXi4uLGf6b7WZeojb8RjTKGV5lh66xpUImkysrusTU5wxldIbsr3gCfXyJ/tQgycpFtyW6xAnH/cFVi9Xr7z8T2hb6/zH5X2dXFZyG/VK2AAAAAElFTkSuQmCC"
                          y="1"
                        />
                      </svg>
                      <p
                        class="sc-kstrdz iUronQ"
                      >
                        Escríbenos por facebook
                      </p>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
            <div
              class="sc-dlfnbm hziHjY"
            >
              <ul
                class="sc-fodVxV cTyRtG"
              >
                <li
                  class="sc-fFubgz jmgknO"
                >
                  <p
                    class="sc-kstrdz iUronQ"
                  >
                    Te buscamos depa
                  </p>
                </li>
                <li
                  class="sc-fFubgz jmgknO"
                >
                  <p
                    class="sc-kstrdz iUronQ"
                  >
                    Blog
                  </p>
                </li>
                <li
                  class="sc-fFubgz jmgknO"
                >
                  <p
                    class="sc-kstrdz iUronQ"
                  >
                    Equipo
                  </p>
                </li>
              </ul>
            </div>
            <div
              class="sc-dlfnbm sc-idOhPF dDGZRQ hEyJIk"
            >
              <p
                class="sc-kstrdz sc-hBEYos iUronQ hDAYlh"
              >
                Aviso de privacidad
              </p>
            </div>
            <div
              class="sc-dlfnbm sc-idOhPF hziHjY hEyJIk"
            >
              <p
                class="sc-kstrdz sc-hBEYos iUronQ hDAYlh"
              >
                Términos y condiciones
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bqyKva jQYkRD"
    >
      <div
        class="sc-bdfBwQ sc-kLgntA guSBMN iXAtuJ"
      >
        <div
          class="sc-gsTCUz ikQfAY"
        >
          <div
            class="sc-dlfnbm eSUTag"
          >
            <div
              class="sc-gsTCUz ikQfAY"
            >
              <div
                class="sc-dlfnbm sc-bkzZxe hziHjY itmDFr"
              >
                <svg
                  class="Footer__Logo-h84djn-5 cnyeeA"
                  height="35"
                  width="35"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M35 0l-.041 35H20.43v-4.503h10.03V4.503H4.58v9.006H0V0h35zM15.69 16.897V35h-4.518v-7.155H4.518V35H0V16.897h4.518v6.853h6.654v-6.853h4.518z"
                    fill="#FF0364"
                  />
                </svg>
              </div>
              <div
                class="sc-dlfnbm fLHlfI"
              >
                <ul
                  class="sc-fodVxV cTyRtG"
                >
                  <li
                    class="sc-fFubgz jmgknO"
                  >
                    <p
                      class="sc-kstrdz iUronQ"
                    >
                      hey@homie.mx
                    </p>
                  </li>
                  <li
                    class="sc-fFubgz jmgknO"
                  >
                    <p
                      class="sc-kstrdz iUronQ"
                    >
                      5015 1546
                    </p>
                  </li>
                  <li
                    class="sc-fFubgz jmgknO"
                  >
                    <svg
                      height="13"
                      width="12"
                      xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink"
                    >
                      <image
                        fill="none"
                        fill-rule="evenodd"
                        height="13"
                        transform="translate(0 -1)"
                        width="12"
                        xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAgCAYAAAAFQMh/AAAABGdBTUEAALGOfPtRkwAABM1JREFUSA3Nll1om2UUx5M3CVk+mtXZimUdrC10Ewdzdl4EhJalNu2uFCbeqExldkOHguKFF1JQwYlUUClWrSgIXlRFJrRpta5ICupa2DoKazFdSLPVdbrZNk2bb38nyxvepG+a2jLwwMl5nvOc8/8/n+eN0bBZ6epS9u055IgpNQ6XKVEVN5srDGaDwRJPRdIJ841kOr4yPffTiqGrK70ZSGO5oJaWFuANTrPZ7DKZTI2KotQbjcaHM5nMfvySP43+kk6nZ1Op1EwymVyiHxkdHU1iS8pGxIrX662E4ACEj4HQhjZAatVDIy6GP4D+SPv7SCRy0e/3L9LX3QFdYla5w2Kx7EdfBORxyFx6ZKV85CyR088OfBiLxaZZ/VpxrKnY0dbW5oDwUba2lzFPqRUW52n7uZwHsY+AM19fX38lEAgktDEFxG632+ZwOE6Q8AFB92gDt9IGZxd6lNyb1dXVk+FwOH/ueWK5RE6n8ymC3id4x1aISuRY8B+xWq1ztbW1l4LBYPbMFTXYZrMd5sa+A6nu5VHjtmIFU7CFQ83PrripqcnOFn9KwAF14A5YO/h7XS7Xt/Pz84nsitn/doiO3AGyAkhuuwcurziF2MRMnkF1n1ZB5u1OCnML1X2fmvg0RCv0JT4rwoE8S8dk5Pns4cpP0qm8Pbzhr4BdRHsBeJVIKSjrJiyEuOcYH6d9lPYuDeotqttBhaok51qWFIAMcTOAPO/z+eSNd9IO5vxZXNpraBjtpXicxtYUkUrcXcKpMCg1d0MhRkgD1OMTAwMD4xI8ODj4M+6TAIewMfQG7W8gPBaPx7sBf4W+Rw8Y/33ccuWQ3mCRLwzpKVbq5wXIu8wK5MMQnqbzA/Y4k3oa0gBv9mN8cmF1hdiDxvb29o8gf0E3AidB1zAnFxYWfBUVFTsBfYD6e4H6+5cmRy5puqOjoxrbw4qOacb0mj2ScF5vJOe7jn2J1Y5VVVU1UwA+YQvP2u32t6h02nuR9ng8d0PYvQlSA3i/K/xcZlX5GiqE9EX+Rl/nBv4hVQfC7wCVz6MN7WQSb6rkYtmJt/E/iZaTFDjTxtbW1p18jSboNGgyopD20F/Gfwp7r2Ys3yTmPSbezcReI+7l/MAGDXICiUSiyTQ7OxtrbGzcS6xbEy8PfzdgclZOjb+46SbmIZxPYNe95+Jg6RP25dDQ0Fk5Y9nzL5jJTWnnpJKAfWqnlBUypFl+SsVo/cIhXOLLEvNMJsntY6DgrLVJ223nsD8XrjyxNKLR6LuY39ByNVjC/6sI5q+rq6tn1MTsZ1E6fKCjDQ0NE1yUFrpSW7O7gd2upFjtNFv83MjIyBUVLE8sDv4XXa+rqxtn2w+jNWrQVi2Ea+Bcoox2Dg8PT2hxdC8FFaiZoHMk6Y5rAfTaufNcYsyHvkFplb+9BSJ/1tcJ2/IPBUM+DHliActNpGCXNMny3Y2hUVkl8X2hUKh/amoqronJN3WJOWft+QqgzP4yKpOpBVT+DKqTykAkn8Or+C5gfcvLy+fGxsaW6ZcUXWI1GhAhnMF+BvjXi4uLGf6b7WZeojb8RjTKGV5lh66xpUImkysrusTU5wxldIbsr3gCfXyJ/tQgycpFtyW6xAnH/cFVi9Xr7z8T2hb6/zH5X2dXFZyG/VK2AAAAAElFTkSuQmCC"
                        y="1"
                      />
                    </svg>
                    <p
                      class="sc-kstrdz iUronQ"
                    >
                      Escríbenos por facebook
                    </p>
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            class="sc-dlfnbm hziHjY"
          >
            <ul
              class="sc-fodVxV cTyRtG"
            >
              <li
                class="sc-fFubgz jmgknO"
              >
                <p
                  class="sc-kstrdz iUronQ"
                >
                  Te buscamos depa
                </p>
              </li>
              <li
                class="sc-fFubgz jmgknO"
              >
                <p
                  class="sc-kstrdz iUronQ"
                >
                  Blog
                </p>
              </li>
              <li
                class="sc-fFubgz jmgknO"
              >
                <p
                  class="sc-kstrdz iUronQ"
                >
                  Equipo
                </p>
              </li>
            </ul>
          </div>
          <div
            class="sc-dlfnbm sc-idOhPF dDGZRQ hEyJIk"
          >
            <p
              class="sc-kstrdz sc-hBEYos iUronQ hDAYlh"
            >
              Aviso de privacidad
            </p>
          </div>
          <div
            class="sc-dlfnbm sc-idOhPF hziHjY hEyJIk"
          >
            <p
              class="sc-kstrdz sc-hBEYos iUronQ hDAYlh"
            >
              Términos y condiciones
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
